Genes for orpha disease_terms are now saved as int, matching the format for omim disease_terms (#4354)
Clinical-Genomics/scout: CHANGELOG.md

- Analysis type and direct link from cases list for OGM cases
- Removed unused `case_obj` parameter from server/blueprints/variant/controllers/observations function
### Fixed

> - All disease_terms have their genes in number format when added to the scout database

Reviewer: ```suggestion
- All disease_terms have gene HGNC ids as integers when added to the scout database
```
